packages/stark-ui/src/modules/collapsible/components/collapsible.component.ts

```diff
@@ -52,12 +75,6 @@ export class StarkCollapsibleComponent extends AbstractStarkUiComponent implemen
 	@Input()
 	public titleLabel: string;
 
-	/**
-	 * Design options related to the component
-	 */
-	@Input()
-	public iconSpinningEnabled: boolean = false;
```

Comment:
Mmm I don't think we should remove this input. What if the user wants to use a custom icon and make it spinning?